### ISSUE TITLE
Wrong output name of .so file when using elixir

### ISSUE THREAD

**Issue by Hermanverschooten** [Feuerlabs/netlink#12]
I added netlink to my `mix.exs` file and compiled it.
When I try to start `iex -S mix` it fails with not finding `netlink_drv`.
If I look into `_build/dev/lib/netlink/priv` there is a file called `netlink.so`, renaming it to `netlink_drv.so` fixes it.
This using `Erlang 21.3.8` with `Elixir 1.8.1-otp-21`, installed with `asdf`

**Comment by Hermanverschooten** [comment]
Hmm, this is probable related to #10.

**Comment by tolbrino** [comment]
> Hmm, this is probable related to #10.

Yes, I'd think so too. Can you verify with the latest `master` branch?
